src/components/HeaderAuthOptions/HeaderSignUp/HeaderSignUp.js

```diff
@@ -62,7 +160,7 @@ class HeaderSignUp extends Component {
           <Form.Group controlId="username" className="mt-4">
             <TextField
               fullWidth={true}
-              className="account-info"
+              className={!usernameVal ? 'account-info-signup-red username input' : 'account-info-signup username input'}
```

Comment:
I'm using the class here to change the color of the form border. Red for invalid and Green for Valid. This will most likely be unnecessary when we implement the new figma drawings.